src/renderer/component/fileRender/view.jsx

```diff
@@ -2,6 +2,7 @@
 import React from 'react';
 import LoadingScreen from 'component/common/loading-screen';
 import PdfViewer from 'component/viewers/pdfViewer';
+import ThreeViewer from 'component/threeViewer';
```

Comment:
I added this so it works, this component should probably live inside of `component/viewers/...` to keep it consistent.